disnake/guild_scheduled_event.py

```diff
@@ -211,15 +211,22 @@ def _update(self, data: GuildScheduledEventPayload) -> None:
         self._image: Optional[str] = data.get("image")
 
     def __repr__(self) -> str:
-        return (
-            "<GuildScheduledEvent "
-            + " ".join(
-                f"{attr}={getattr(self, attr)!r}"
-                for attr in self.__slots__
-                if not attr.startswith("_")
-            )
-            + ">"
+        attrs = (
+            ("id", self.id),
+            ("guild_id", self.guild_id),
+            ("channel_id", self.channel_id),
+            ("name", self.name),
+            ("description", self.description),
+            ("scheduled_start_time", self.scheduled_start_time),
+            ("scheduled_end_time", self.scheduled_end_time),
+            ("privacy_level", self.privacy_level),
+            ("status", self.status),
+            ("entity_type", self.entity_type),
+            ("entity_metadata", self.entity_metadata),
+            ("creator", self.creator),
```

Comment:
do we intend to be missing created_at? Not sure.

Reply:
Don't think so, I'm not seeing any other `__repr__`s that reflect the `created_at` value